src/modules/core/components/Fields/SelectOption/SelectOption.css

```diff
@@ -4,23 +4,27 @@
   font-weight: var(--weight-bold);
   text-overflow: ellipsis;
   white-space: nowrap;
-  color: var(--grey-1);
+  color: var(--dark);
   cursor: pointer;
+}
 
-  &[aria-selected="true"] {
-    background-color: var(--grey-blue-0);
-    outline: none;
-  }
+.main[aria-selected="true"] {
+  background-color: var(--grey-blue-0);
+  outline: none;
+}
 
-  &[aria-checked="true"] {
-    color: var(--primary);
-    pointer-events: none;
-  }
+.main[aria-checked="true"] {
+  color: var(--primary);
+  pointer-events: none;
+}
+
+.main[aria-disabled="true"] {
+  color: color-mod(var(--dark) alpha(50%));
+  pointer-events: none;
+}
 
-  &[aria-disabled="true"] {
-    color: color-mod(var(--grey-1) alpha(50%));
-    pointer-events: none;
-  }
+.main[data-checked="true"] {
+  color: var(--primary);
```

Comment:
Why was this introduced? 

This colors all checkboxes default selected options as "green" (our "primary" color)

![Screenshot from 2021-02-05 14-35-33](https://user-images.githubusercontent.com/1193222/107037244-7a1df780-67c3-11eb-901c-f68a61087734.png)

![Screenshot from 2021-02-05 14-35-37](https://user-images.githubusercontent.com/1193222/107037247-7ab68e00-67c3-11eb-9453-1677126a549c.png)

![Screenshot from 2021-02-05 14-36-21](https://user-images.githubusercontent.com/1193222/107037343-9b7ee380-67c3-11eb-9a7c-15fafd042ba7.png)


If this was done for a single specific select box then please refactor this, otherwise, I don't think this is a good change.

Reply:
This was introduced due to this: `Selected domain colour (dropdown) is wrong (consistent behaviour)` being in vol.1 of the feedback provided by Karol and Jack. If I remember correctly, I checked the style guide in Figma, and I saw that the selected option colour is the primary colour.

Do you think it's better in the old colour?

Reply:
Nah leave it, don't bother.